From 59ea66a4d4899f682fbbf6c613532f05dbb69059 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Wed, 20 Jan 2021 12:08:26 -0800
Subject: [PATCH 1/2] Set CMake flags for Fast-RTPS

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 rpm/template.spec.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index 3464a0bc..91b8a7fd 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -53,6 +53,8 @@ mkdir -p .obj-%{_target_platform} && cd .obj-%{_target_platform}
 %if !0%{?with_tests}
     -DBUILD_TESTING=OFF \
 %endif
+    -DINSTALL_EXAMPLES=OFF \
+    -DSECURITY=ON \
     ..
 
 %make_build
-- 
2.43.0

